static/js/stream_list.js

```diff
@@ -461,9 +461,10 @@ exports.initialize = function () {
         if (overlays.is_active()) {
             ui_util.change_tab_to('#home');
         }
-        var stream = $(e.target).parents('li').attr('data-name');
+        var stream_id = $(e.target).parents('li').attr('data-stream-id');
+        var stream = stream_data.get_sub_by_id(stream_id);
```

Comment:
Hmm.  I've been working to purge use of `stream` and replace it with `stream_name` wherever I see it, but perhaps we should really finish that before starting to use `stream` for an object.